recipes/exejs/meta.yaml

```diff
@@ -16,19 +16,19 @@ build:
 
 requirements:
   host:
-    - python {{ python_min }}
-    - node.js
+    - python >={{ python_min }}
     - setuptools
     - wheel
     - pip
   run:
-    - python >=3.8
-
+    - python >={{ python_min }}
+    - nodejs
 test:
   imports:
     - exejs
   commands:
     - pip check
+    - node --version
```

Comment:
If your package has a CLI test, add that here. You don't need to test conda-forge's node.
```suggestion
```